.github/workflows/utils-workflow.yml

```diff
@@ -1,69 +0,0 @@
-name: Pytests
```

Comment:
The behavior tested by this workflow is now tested in `isofit/test/test_cli_machinery.py`.